src/services/TestDataService.ts

```diff
@@ -2157,4 +2158,44 @@ export class TestDataService {
 
         return Object.assign({}, basicCmsPage, overrides);
     }
+
+    getBasicCustomerGroup(overrides: Partial<CustomerGroup>): Partial<CustomerGroup> {
```

Comment:
```suggestion
    getBasicCustomerGroup(overrides: Partial<CustomerGroup> = {}): Partial<CustomerGroup> {
```

make it here also as optional parameter.